samples/bluetooth/mesh_demo/README.rst

```diff
@@ -0,0 +1,54 @@
+.. _ble_mesh_demo:
+
+Bluetooth: Mesh Demo
+####################
+
+Overview
+********
+
+A Bluetooth Mesh application primarily intended for demonstration
+purposes. The application provisions and configures itself (i.e. no
+external provisioner needed) with hard-coded network and application key
+values. The local unicast address can be set using a NODE_ADDR build
+variable (e.g. NODE_ADDR=0x0001 for unicast address 0x0001), or by
+manually editing the value in the board.h file.
+
+Because of the hard-coded values the application is not suitable for
+production use, but is quite convenient for quick demonstrations of Mesh
+functionality.
+
+The application has some features especially designed for the BBC
+micro:bit boards, such as the ability to send messages using the board's
+buttons as well as showing information of received messages on the
+board's 5x5 LED display. It's generally recommended to use unicast
+addresses in the range of 0x0001-0x0009 for the micro:bit since these
+map nicely to displayed addresses and the list of destination addresses
+which can be cycled with a button press. A special address, 0x000f, will
+make the application become a heart-beat publisher, and enable the other
```

Comment:
change to ``enables``

Reply:
That doesn't sound right to me. It's basically "will make X, and enable Y", i.e. isn't it still riding on "will", meaning it should be "will enable" instead of "will enables"?

Reply:
Ah, true.  I parse the sentence a bit differently because of the comma.  How about this:

    A special address, 0x000f, will make the application become a heart-beat publisher 
    and enable the other nodes to show information of the received heartbeat messages.